test/elixir/test/bulk_docs_test.exs

```diff
@@ -130,6 +130,17 @@ defmodule BulkDocsTest do
     assert Enum.at(rows, 2)["error"] == "conflict"
   end
 
+  @tag :with_db
+  test "bulk docs raises error for transaction larger than 10MB", ctx do
+    docs = [%{_id: "0", a: random_string(16_000_000)}]
+    old_size = Couch.get("/_node/node1@127.0.0.1/_config/couchdb/max_document_size").body
+    set_config_raw("couchdb", "max_document_size", "67108864") # 64M
+    resp = Couch.post("/#{ctx[:db_name]}/_bulk_docs", body: %{docs: docs})
+    set_config_raw("couchdb", "max_document_size", old_size) # set back
+    assert resp.status_code > 399
```

Comment:
Wonder what the actual status code is, should we check for it specifically?

Reply:
It's a 500, see https://github.com/apache/couchdb/pull/3355#pullrequestreview-585060716 . I just didn't feel comfortable persisting that we "expect" a 500.

Reply:
It may be better to be precise if we can. There might be some advantage to have the test break if we ever cap the max_document_size config based on FDB limits.

Reply:
Done in accb8f1e3 .
The original idea was: let's not assert on a "product bug". We only want to know that it fails (`>399`), it might actually be a "user error" (`4xx`).
But the fact that I'm asserting on the `reason` that has an FDB status code kind of "allows" us to assert on an 500 here.